similarity/src/main/java/edu/illinois/cs/cogcomp/llm/comparators/WordComparator.java

```diff
@@ -0,0 +1,110 @@
+/**
+ * This software is released under the University of Illinois/Research and Academic Use License. See
+ * the LICENSE file in the root folder for details. Copyright (c) 2016
+ *
+ * Developed by: The Cognitive Computation Group University of Illinois at Urbana-Champaign
+ * http://cogcomp.cs.illinois.edu/
+ */
+package edu.illinois.cs.cogcomp.llm.comparators;
+
+import java.io.IOException;
+
+import edu.illinois.cs.cogcomp.config.SimConfigurator;
+import edu.illinois.cs.cogcomp.core.utilities.configuration.ResourceManager;
+import edu.illinois.cs.cogcomp.nlp.utilities.StringCleanup;
+import edu.illinois.cs.cogcomp.sim.Metric;
+import edu.illinois.cs.cogcomp.sim.MetricResponse;
+import edu.illinois.cs.cogcomp.sim.NESim;
+import edu.illinois.cs.cogcomp.sim.WordSim;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import edu.illinois.cs.cogcomp.mrcs.comparators.Comparator;
+import edu.illinois.cs.cogcomp.mrcs.dataStructures.EntailmentResult;
+
+/**
+ * 
+ * @author mssammon
+ *
+ */
+public class WordComparator implements Comparator<String, EntailmentResult> {
+	private boolean computeSimpleScore;
+	private double entailmentThreshold;
+	protected boolean defaultUpwardMonotone = true;
+
+	private Metric wordSim;
+
+	private Logger logger = LoggerFactory.getLogger(WordComparator.class);
+
+	// private LlmConstants.WordMetric metric;
+
+	public WordComparator(String configFile_) throws IOException {
+		ResourceManager rm = new ResourceManager(configFile_);
+		configure(rm);
+	}
+
+	public WordComparator(ResourceManager rm_) throws IllegalArgumentException, IOException {
+		configure(rm_);
+	}
+
+	public WordComparator() throws IllegalArgumentException, IOException {
+		wordSim = new NESim();
```

Comment:
This should use WNSim, not NESim, as default. Please use the appropriate Configurator object to set default values.